packages/cli-kit/src/node/cli.ts

```diff
@@ -46,7 +47,18 @@ export async function runCLI(options: RunCLIOptions) {
 
   run(undefined, options.moduleURL)
     .then(flush)
-    .catch((error: Error): Promise<void | Error> => {
+    .catch(async (error: Error): Promise<void | Error> => {
+      if (error instanceof CancelExecution) {
+        info(
+          `✨  ${
```

Comment:
I don't think we need to print anything else here. Also, we are already showing the `Done in X seconds` message, so that's enough IMO:

![image](https://user-images.githubusercontent.com/14979109/177983623-a51634c1-4b78-480d-9113-cbdda8fb80f0.png)

Reply:
I think you are right. We could leave the option to print the exception message only in case the developer add it to it when it is thrown

Reply:
Nope:

![11-10-ufvyp-vczsf](https://user-images.githubusercontent.com/14979109/178271704-19c49f14-45c6-4e9c-b2fd-4d852e3cbd04.png)

Reply:
Ah, right... I thought it was oclif. Anyway, I think there's no need to show anything else in this case.